requirements/requirements-neptune.txt

```diff
@@ -1 +1 @@
-neptune-client
+neptune
```

Comment:
This needs to specify version >= 0.16.17 for `stringify_unsupported` and `.append`.

Reply:
I think this should be non-issue as `neptune` package is available only from 1.0 onwards, so both should be available.

Ref: https://pypi.org/project/neptune/#history

Reply:
Ok, but then we don't need the `try ... except` block. If we say that we support v < 1.0.0, we should somewhere make it clear that it's only 0.16.17 or 0.16.18.

Reply:
> Ok, but then we don't need the try ... except block.

try .. except is to support the previous `neptune-client` versions (0.16.17 onwards). Although, I have tested the older versions locally, I am not sure if we can test them easily in CI. So I have opted for testing the latest `neptune`.

>  If we say that we support v < 1.0.0, we should somewhere make it clear that it's only 0.16.17 or 0.16.18.

Hmmm, I thought we want to promote use of `neptune` but work with existing installation (0.16.17 and 0.16.18) if we can.

@AleksanderWWW what is your opinion?